ruby/vampires.rb

```diff
@@ -72,3 +72,8 @@
 end
 
 puts "Actually, never mind! What do these questions have to do with anything? Lets all be friends!"
+
+#requesting code review
```

Comment:
In the future, include the full file as a change in the PR so I can comment in-line with the code.